docs/operation/bootstrap_backup_restore.rst

```diff
@@ -0,0 +1,49 @@
+Bootstrap node backup restore
+=============================
+This section describes a reliable way for backup and restore a **MetalK8s**
```

Comment:
This section describes how to backup a **MetalK8s** bootstrap node and how to restore a bootstrap node from such backup.